src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -66,6 +67,9 @@ window.qBittorrent.DynamicTable ??= (() => {
 
     let DynamicTableHeaderContextMenuClass = null;
 
+    if (typeof LocalPreferences === "undefined")
+        window.LocalPreferences = new window.qBittorrent.LocalPreferences.LocalPreferences();
```

Comment:
I think this should be a local variable. It is quite messy to inject into global IMO.

Reply:
This file already assumes there is a declared and initialized variable named `LocalPreferences`, which effectively means it expects this global variable to exist. Without this change, the table fails to initialize in the Add Torrent frame because `LocalPreferences` isn't defined.

Reply:
>This file already assumes there is a declared and initialized variable named LocalPreferences, which effectively means it expects this global variable to exist.

Right.
However, taking a step back. It seems to me that `LocalPreferences` don't really require to be global. I mean it should be preferred to be local.
If this is too complex to change in this PR (which I hope it is not) then it can be deferred later.